### DISCUSSION TITLE
Change column colour

### DISCUSSION THREAD

**Discussion by nk-tedo-001** [eza-community/eza#987] [Q&A, Howto, Help] [answered]
Hey!

The date column is too dark for me 
<img width="688" alt="image" src="https://github.com/eza-community/eza/assets/144455088/3ef998d4-ec8b-448a-87b9-00cb343817e5">
 
Is it possible to override its colour?

**Comment by gierens** [comment, marked as answer]
Yes, you can use the environment variable `EZA_COLORS`, for example:

![Screenshot from 2024-05-23 08-13-39](https://github.com/eza-community/eza/assets/49617392/d8c03261-8a1f-44b5-9ed4-81fc9b70132d)

The man page `eza_colors` contains more information on this, what variables exist and the different color codes. So configure this the way you want in your shell profile.

Note: we are currently also working on a config file feature, see #840
